### PR TITLE
fix #138: allow a single modifier as accelerator value

### LINKED ISSUE #138: Allow accelerator to use meta-only keys as an entry/shortcut.
**Is your feature request related to a problem? Please describe.**
A user opened this ticket in my app, requesting to be able to use `alt` as a shortcut by itself.
https://github.com/lacymorrow/crossover/issues/129

**Additional context**
When I setup the accelerator, I allowed single-key shortcuts to be used (unless the `modifierRequired` property is true), however that does not work with only modifiers (`shift`, `alt`, `control`, `command`).

I'd like to add a property to the accelerator field that allows modifiers to be used as single-key shortcuts.

### PR DESCRIPTION
**Pull request by lacymorrow**
Basically allows a modifier like `Alt` to be set by itself.

I have a feature that folks would like to bind to individual keys, including modifiers.

### CODE REVIEW

**Comment by lacymorrow** [comment]
cc @pvrobays

**Comment by pvrobays** [comment]
@lacymorrow Meanwhile I've merged the cleanup PR where the readme file has changed quite a lot (because of formatting). Can you fix the conflicts?

File: src/app/components/main/components/group/components/fields/accelerator/index.jsx [hunk 1]
```diff
@@ -23,19 +24,34 @@ const AcceleratorField = ({ field, value, onChange, key }) => {
 		like control, shift, etc. as well as a handful of oddball codes.
 		We only save if we receive one of these non-modifier keys.
 	*/
-	// const modifierKeyCodes = [16, 17, 18, 91, 92, 93]
+	const modifierKeyCodes = [ 16, 17, 18, 91, 92, 93 ];
 	const specialKeyCodes = new Set([ 0, 1, 2, 3, 4, 5, 6, 7, 10, 11, 16, 17, 18, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 91, 92, 93, 94, 95 ]);
 
+	// Assign modifier key names, different OS' use different key names
+	let altKeyName = 'Alt';
+	let metaKeyName = 'Meta';
```
[the remark this answers is not in the document]
Comment (lacymorrow) [reply src/app/components/main/components/group/components/fields/accelerator/index.jsx:32]: It varies by region and layout, but for linux users 'meta' or 'super' work (and are recognized by electron)

**Comment by lacymorrow** [comment]
done

**Comment by pvrobays** [comment]
Thanks!
Sorry for the delay. I just became a father :)

**Comment by lacymorrow** [comment]
Yooooo, CONGRATULATIONS!!! 🎉 🎁 👶  
Enjoy it man, take care of your family and remember to sleep

**Pull request merged**